packages/zosfiles/src/api/methods/hRecall/doc/IRecallOptions.ts

```diff
@@ -9,14 +9,12 @@
 *
 */
 
+import { IMigrateOptions } from "../../hMigrate";
+
 /**
- * This interface defines the options that can be sent into the copy data set function.
+ * This interface defines the options that can be sent into the recall data set function.
  */
-export interface IRecallOptions {
 
-  /**
-   * If true then the function waits for completion of the request.
-   * If false the request is queued.
-   */
-  wait?: boolean;
+// tslint:disable-next-line: no-empty-interface
+export interface IRecallOptions extends IMigrateOptions{
```

Comment:
Refer to comment on extending `IMigrateOptions`, is there a better way to implement this?

Reply:
@zFernand0 The `//tslint:disable-next-line: no-empty-interface` is what made me think that there could be a better implementation as effectively this is just another file when all it does is use the same options as `IMigrateOptions`. The `IZosFilesUtils` you suggested would be good to look into, has something like a `IZosfilesUtils` been done before?